source/api/auth/0.9/implementation.md

```diff
@@ -57,46 +57,47 @@ At a minimum, a client implementation following a "happy path" would:
 
 There are different ways of doing implementing this in different styles of client user interface. These notes explore the issues implementors will encounter.
```

Comment:
"implementors" -> "implementers"  http://dictionary.cambridge.org/us/dictionary/english/implementer (Also in TOC and section 7 header)